models/enterprise_sonic/ntp/deleted_example_03.txt

```diff
@@ -0,0 +1,10 @@
+# Using deleted
+#
+
```

Comment:
Please add "before" and "after" state displays for this example. If there is no appropriate NTP "show" command to display it, please use "show running-configuration" and show just the relevant NTP part of the output.

e.g.

```
show running-configuration
                *
                *
                *
ntp authenticate
ntp authentication-key 80 md5 U2FsdGVkX1/X2M9RtUU5O6HHrLbG7RH0+nSkz4j4HQg= encrypted
ntp authentication-key 90 md5 U2FsdGVkX18PBecDvwvxBpbgG/k8xZzphnUYfQW4ktE= encrypted
                *
                *
                *

Reply:
Added "before" and "after" states